packages/api-kit/tests/e2e/getSafeOperationsByAddress.test.ts

```diff
@@ -1,4 +1,4 @@
-import SafeApiKit from '@safe-global/api-kit'
+import SafeApiKit from '@safe-global/api-kit/index'
```

Comment:
Is it mandatory to add the `index`?

Reply:
Without it, I see this weird error:
![Screenshot 2024-06-20 at 17 12 36](https://github.com/safe-global/safe-core-sdk/assets/4171783/8ab6654d-0f49-4ace-8efd-e6604dafd2cc)

All the other files in the same folder are defining the same import path for the SafeApiKit. That's why I changed it accordingly here.